AGPs/AGP-LD1.md

```diff
@@ -0,0 +1,63 @@
+
+At the moment there is no incentive to do this outside of one-off grants. There has been discussion to create monetization models for the eventual release of the App Center, but currently such a system doesn't yet exist, and even if it did the number of users of Aragon is small enough that such a market may not be attractive enough to bring top talent into the space. Users will come if there are high quality apps, but if there are no users, no high quality apps will be built and maintained.
+
+To remedy this I propose we bootstrap the supply side by allocating funds and distributing them to Aragon Application developers based on agreed upon KPIs. We can combine multiple KPIs into an **Aragon App Score**, which we can use to rank applications and then use the ranking to distribute funding to maintainers on a periodic basic.
+
+**The following weighted KPIs would be used to create the Aragon App Score:**
+
+| KPI | Formula | Weight |
+| -------- | -------- | -------- |
+| Utility    | Sum of (# of transactions involving an application ) | 30%  |
+| Value weighted utility    | Sum of (# of transactions involving an application * DAI value of organization's treasury) | 30%    |
+| Popularity    | Sum of (# of installations) | 10%  |
+| Value weighted Popularity    | Sum of (# of installations *  DAI value of organization's treasury) | 30% |
+
+To compute the DAI value of an organizations treasury we would use the *default vault* and compute the DAI value of ETH, ANT, and DAI held by the organization.
+
+To compute the # of transactions involving an application we would simply examine the blockchain.   
+
+For each payout cycle, there is a determined “pot” of total earnings that will get paid to apps. The top app gets paid 20% of the total pot. So, for a pot of 100K ANT, the top app receives 20K ANT. The next app gets paid 20% of the remaining pot. The remaining pot is 80K, and 20% of that is 16K ANT. This process continues until every app has been paid or the payout amount is bellow 200 ANT whichever comes first.
+
+**Blacklist**
+
+It's possible that the KPI mechanism can be gamed, for example the popularity metric could be manipulated by creating a bunch of organization and installing a bunch of otherwise unpopular applications. For the purposes of our initial pilot, rather than try an engineer un-gameable metric, we propose that we can rely on the community to report potential abuse and the discretion of Pinion to manage a blacklist for the purpose of excluding recipients that are suspected to be trying to maliciously manipulate payout amounts. In the future, this role could be decentralized by leveraging the Aragon Court.
```

Comment:
```suggestion
It's possible that the KPI mechanism can be gamed, for example the popularity metric could be manipulated by creating a bunch of organizations and installing a bunch of otherwise unpopular applications. For the purposes of our initial pilot, rather than try to engineer an un-gameable metric, we propose that we can rely on the community to report potential abuse and the discretion of Pinion to manage a blacklist for the purpose of excluding recipients that are suspected to be trying to maliciously manipulate payout amounts. In the future, this role could be decentralized by leveraging the Aragon Court.
```